projects/core/src/configurator/commons/facade/configurator-group-status.service.ts

```diff
@@ -48,6 +48,22 @@ export class ConfiguratorGroupStatusService {
     );
   }
 
+  /**
+   * Returns the first non-conflict group of the configuration which is not completed
+   * and undefined if all are completed.
+   *
+   * * @param configuration - Configuration
```

Comment:
Remove ***** (see line 55)
Add **@return** with the description what is expected to be returned